### PR TITLE
[Feature] Allow QR codes with unlimited duration

### LINKED ISSUE #12: [Feature] Allow QR codes with unlimited duration
As a user, I'd like to create QR codes that don't get deleted automatically. To be able to delete them manually, I'd like an admin view that allows me to delete them at a later time.

### PR DESCRIPTION
**Pull request by nwittstruck**
Closes #12 

This PR adds:
- [x] The creation of QR codes with unlimited duration. They will not be deleted automatically by the worker.
- [x] Adds an admin link that is shown just after QR code creation. Using this admin link it is possible to delete QR codes manually later. This admin link is shown for all QR codes of the types audio and text.

### CODE REVIEW

File: lib/qrstorage_web/templates/qr_code/download.html.heex [hunk 1]
```diff
@@ -2,7 +2,17 @@
     <div id="canvas" data-color={@qr_code.color} data-url={@qr_code.text}></div>
 <% else %>
     <div id="canvas" data-color={@qr_code.color} data-url={Routes.qr_code_url(@conn, :show, @qr_code)}></div>
+    <%= if get_flash(@conn, :admin_url_id) do %>
```
Comment (gerardo-navarro) [inline lib/qrstorage_web/templates/qr_code/download.html.heex:5]: Just to be sure. It is intended that qr codes with type `:link` cannot be deleted. I would have expected to work for link qr codes as well

File: lib/qrstorage_web/controllers/qr_code_controller.ex [hunk 2]
```diff
@@ -28,6 +28,7 @@ defmodule QrstorageWeb.QrCodeController do
         end
 
         conn
+        |> put_flash(:admin_url_id, qr_code.admin_url_id)
```
Comment (gerardo-navarro) [inline lib/qrstorage_web/controllers/qr_code_controller.ex:31]: I understand why you put the `admin_url_id` in the flash section. However, I think including this as a param in the redirect feels cleaner instead using the workaround via `put_flash(...)`

**Review by gerardo-navarro — APPROVED** [review]
There are just some small remarks from my side. A part from this, the code and function look good.

File: lib/qrstorage_web/templates/qr_code/download.html.heex (hunk 1 above)
Comment (nwittstruck) [reply lib/qrstorage_web/templates/qr_code/download.html.heex:5]: This is fine, they get deleted automatically after one hour. We just need to store them to show the QR code.

File: priv/repo/migrations/20220218141645_add_admin_uuid.exs [hunk 3]
```diff
@@ -0,0 +1,19 @@
+defmodule Qrstorage.Repo.Migrations.AddAdminUuid do
```
[the remark this answers is not in the document]
Comment (nwittstruck) [reply priv/repo/migrations/20220218141645_add_admin_uuid.exs:1]: Yes, I thought the same. Unfortunately the order of adding/removing the crypto extension needs to be different for the up / down migrations. That's why it has to happen in one step.

**Pull request merged**